src/mmrelay/meshtastic_utils.py

```
@@ -870,8 +884,21 @@ def on_meshtastic_message(packet, interface):
             return
 
         # Attempt to get longname/shortname from database or nodes
-        longname = get_longname(sender)
-        shortname = get_shortname(sender)
+        try:
+            longname = get_longname(sender)
+        except Exception:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)